modules/images/webp-uploads/load.php

```diff
@@ -573,3 +584,47 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
 	return webp_uploads_img_tag_update_mime_type( $html, 'post_thumbnail_html', $attachment_id );
 }
 add_filter( 'post_thumbnail_html', 'webp_uploads_update_featured_image', 10, 3 );
+
+/**
+ * Adds a fallback mechanism to replace webp images with jpeg alternatives on older browsers.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_wepb_fallback() {
+	// Get mime type transofrms for the site.
+	$transforms = webp_uploads_get_upload_image_mime_transforms();
+
+	// We need to add fallback only if jpeg alternatives for the webp images are enabled for the server.
+	$preserve_jpegs_for_jpeg_transforms = isset( $transforms['image/jpeg'] ) && in_array( 'image/jpeg', $transforms['image/jpeg'], true ) && in_array( 'image/webp', $transforms['image/jpeg'], true );
+	$preserve_jpegs_for_webp_transforms = isset( $transforms['image/webp'] ) && in_array( 'image/jpeg', $transforms['image/webp'], true ) && in_array( 'image/webp', $transforms['image/webp'], true );
+	if ( ! $preserve_jpegs_for_jpeg_transforms && ! $preserve_jpegs_for_webp_transforms ) {
+		return;
+	}
+
+	$fallback_url = plugins_url( '/fallback.js', __FILE__ );
+
+	$script = <<<EOL
+( function( d, i, s, p ) {
+	s = d.createElement( s );
+	s.src = '{$fallback_url}';
+
+	i = d.createElement( i );
+	i.src = p + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
+	i.onload = function() {
+		i.src = p + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
+	};
+
+	i.onerror = function() {
+		d.body.appendChild( s );
+	};
+} )( document, 'img', 'script', 'data:image/webp;base64,UklGR' );
+EOL;
+
+	wp_print_inline_script_tag(
+		preg_replace( '/\s+/', '', $script ),
+		array(
+			'id'            => 'webpUploadsFallbackWebpImages',
+			'data-rest-api' => trailingslashit( get_rest_url() ),
```

Comment:
```suggestion
			'data-rest-api' => sanitizie_url( trailingslashit( get_rest_url() ) ),
```

`wp_print_inline_script_tag()` does escape attributes with `esc_attr()` -- this ensures it's a URL (basically `esc_attr( sanitize_url() ) === esc_url()`.

Reply:
Updated.